src/ProgressOnderwijsUtils/Data/QueryScalarParameterComponent.cs

```diff
@@ -15,8 +15,12 @@ public void ToSqlParameter(ref SqlParamArgs paramArgs)
         public static void AppendScalarParameter<TCommandFactory>(ref TCommandFactory factory, [CanBeNull] object o)
             where TCommandFactory : struct, ICommandFactory
         {
-            var param = new QueryScalarParameterComponent { EquatableValue = o ?? DBNull.Value };
-            ParameterizedSqlFactory.AppendSql(ref factory, factory.RegisterParameterAndGetName(param));
+            if (o == null || o == DBNull.Value) { 

```

Comment:
Previously, I did this in generic edit, but it's probably better to just do it for all queries.

Conceivably this too might cause queries to break.

However, running our test set locally doesn't show anything dramatic, in any case.